### PR TITLE
Logout Feature

### PR DESCRIPTION
**Pull request by aniansson**
## Changes proposed in this pull request:
Made features and ran tests for the logout page.

## What I have learned working on this feature: 
We have started to understand how to be selfgoing and write these test on our own. We have learnt more about helpers and routes. Although we did need some guidance we feel more secure.

Ready for review!

### CODE REVIEW

File: features/step_definitions/logout_steps.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Given(/^that I am logged in as "([^"]*)"$/) do |name|
+  user = User.create(name: name)
+  login_as(user, scope: :user)
+end
```
Comment (AmberWilkie) [inline features/step_definitions/logout_steps.rb:5]: Ok so what happens if we want to re-use this step, but our users already exists? We should break these into two things. `Given the following users exist` or just `Given there is a user with name "Test-User"` and then the login. That way we can use *both* of those actions independently in future tests.

File: features/step_definitions/logout_steps.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+end
+
+Then(/^I should see "([^"]*)"$/) do |text|
+  expect(page).to have_content text
+end
+
+# Given(/^show me the page$/) do
+#   save_and_open_page
+# end
```
Comment (AmberWilkie) [inline features/step_definitions/logout_steps.rb]: You don't have to comment this out, you can leave it here waiting to be re-used the next time we need to view the page.

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  #     --seed 1234
+  config.order = :random
+
+  # Seed global randomization in this process using the `--seed` CLI option.
+  # Setting this allows you to use `--seed` to deterministically reproduce
+  # test failures related to randomization by passing the same `--seed` value
+  # as the one that triggered the failure.
+  Kernel.srand config.seed
+=end
+end
```
Comment (AmberWilkie) [inline spec/spec_helper.rb:99]: Delete comments out of this file or Thomas will be annoyed.

**Review by AmberWilkie — CHANGES_REQUESTED** [review]
Good work! The next round of testing should be easier for you.

File: features/step_definitions/logout_steps.rb (hunk 1 above)
Comment (tochman) [reply features/step_definitions/logout_steps.rb:5]: Use FactoryGirl please

**Review by AmberWilkie — APPROVED** [review]

**Pull request merged**